cmd/copy.go

```diff
@@ -369,12 +369,6 @@ func (raw rawCopyCmdArgs) cook() (cookedCopyCmdArgs, error) {
 			cooked.pageBlobTier != common.EPageBlobTier.None() {
 			return cooked, fmt.Errorf("blob-tier is not supported while copying from sevice to service")
 		}
-		// Disabling blob type override.
-		// i.e. not support block -> append/page, append -> block/page, page -> append/block,
-		// and when file and s3 is source, only block blob destination is supported.
-		if cooked.blobType != common.EBlobType.None() {
```

Comment:
Removing this enables Azure Files->Blob with a blob type override.  We don't have tests for that. So we need to either add tests for it, or else have this code fail the job `if cooked.FromTo == common.EFromTo.FileBlob() && cooked.blobType != common.EBlobType.None() `.
IMHO it's fine to just prevent it with the above test, rather than enabling it, since the requirement at present is only Page <-> Block

Reply:
Likewise, and perhaps more significantly, removing this enables S3 -> Page blob and S3 -> Append blob. So need to either add tests for that to our existing test suite (possibly in the tests that are implemented in Python, if that happens to be the only place where ew test from S3) OR have this code still fail the job if FromTo == S3Blob
IMHO it's fine to just prevent it with the above test, rather than enabling it, since the requirement at present is only Page <-> Block

Reply:
If you resolve the above by actually supporting the combinations (i.e. the "add tests" option), you'll need to remove the word Block from these two lines in cmd/helpMessages.go

```
  - Azure File (SAS) -> Azure Block Blob (SAS or OAuth authentication)
  - AWS S3 (Access Key) -> Azure Block Blob (SAS or OAuth authentication)
```

Reply:
Agreed. We should be careful and not enable options that we haven't tested yet. Given our scale, users will find out the unintended features within a week.